service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java

```diff
@@ -195,16 +195,11 @@ private boolean needSetFrameworkVersion() {
     if (serviceCenterInfo == null) {
       LOGGER.warn("Server startup when service center not started and cannot retrieve version info, assume latest.");
       return true;
-    } else {
-      Version scVersion = VersionUtils.getOrCreate(serviceCenterInfo.getVersion());
-      Version frameworkVersion = VersionUtils.getOrCreate(Const.SERVICECENTER_FRAMEWORK_VERSION);
-
-      if (scVersion.compareTo(frameworkVersion) >= 0) {
-        return true;
-      } else {
-        return false;
-      }
     }
+
```

Comment:
This change doesn't relate to this JIRA, we need to avoid mixing the unrelated changes together.